plugins/kv-store/package.json

```
@@ -0,0 +1,82 @@
+{
+  "name": "@flatfile/plugin-kv-store",
+  "version": "0.0.1",
```

Start this at `0.0.0`. The changeset will increment it to `0.0.1`

```suggestion
  "version": "0.0.0",
```

---

just run `npm run changeset`